.github/workflows/ci.yml

```diff
@@ -77,7 +49,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: ["2.0", "ruby"]
+        ruby: ["2.5", "ruby"]
```

Comment:
🗒️ Run slow functional tests on the oldest Ruby we support (2.5) and the latest Ruby.